.pipelines/npm/npm-cni-integration-test.yaml

```diff
@@ -120,7 +120,8 @@ jobs:
           --provider=local \
           --ginkgo.focus="$focus" \
           --ginkgo.skip="NetworkPolicyLegacy|SCTP" \
-          --kubeconfig=$HOME/.kube/config
+          --kubeconfig=$HOME/.kube/config \
+          --ginkgo.timeout="2h"
```

Comment:
Noticing we're only setting timeout in this file. Just curious what is the default timeout?

Reply:
60m or 1h. Had to set this as the update to the repo enforced some auto timeouts within ginkgo.